src/nanoemoji/write_font.py

```diff
@@ -148,24 +160,30 @@ class ColorGenerator(NamedTuple):
         ".ttf",
     ),
     "cbdt": ColorGenerator(
-        lambda *args: _not_impl("apply_ufo", "cbdt", *args),
-        lambda *args: _not_impl("apply_ttfont", "cbdt", *args),
+        lambda *args: None,
+        lambda *args: _cbdt_ttfont(*args),
         ".ttf",
     ),
     "sbix": ColorGenerator(
-        lambda *args: _not_impl("apply_ufo", "sbix", *args),
-        lambda *args: _not_impl("apply_ttfont", "sbix", *args),
+        lambda *args: None,
+        lambda *args: _sbix_ttfont(*args),
         ".ttf",
     ),
-    # https://github.com/googlefonts/nanoemoji/issues/260 svg & colr; max compatibility
+    # https://github.com/googlefonts/nanoemoji/issues/260 svg, colr
     # Meant to be subset if used for network delivery
     "glyf_colr_1_and_picosvgz": ColorGenerator(
         lambda *args: _colr_ufo(1, *args),
         lambda *args: _svg_ttfont(*args, picosvg=True, compressed=True),
         ".ttf",
     ),
+    # https://github.com/googlefonts/nanoemoji/issues/260 svg, colr, sbix; max compatibility
+    # Meant to be subset if used for network delivery
+    "glyf_colr_1_and_picosvgz_and_sbix": ColorGenerator(
```

Comment:
Per f2f w/Cosimo we should be uncompressed svg and cbdt because sbix is wildly inconsistently implemented. See https://github.com/harfbuzz/harfbuzz/issues/2679.

Reply:
Fixed

Reply:
Fixed